### PR TITLE
Add homepage content

### PR DESCRIPTION
**Pull request by koslib**

### CODE REVIEW

File: static/.DS_Store
Comment (dreamPathsProjekt) [inline static/.DS_Store:1]: Please remove and add to .gitignore

File: resources/_gen/assets/scss/css/base.scss_3b33337114e481782feeb60752452e17.json
Comment (dreamPathsProjekt) [inline resources/_gen/assets/scss/css/base.scss_3b33337114e481782feeb60752452e17.json:1]: Are these css fingerprinted files need to be git tracked? They always update fingerprints on any build (due to invalidate caching)

**Review by dreamPathsProjekt — CHANGES_REQUESTED** [review]
Have some doubts about git tracking. Not content.

File: static/.DS_Store
Comment (koslib) [reply static/.DS_Store:1]: Apologies - fixed in 370cfab8fe1434c04878c8ddf2357c6279f54ffc

File: resources/_gen/assets/scss/css/base.scss_3b33337114e481782feeb60752452e17.json
Comment (koslib) [reply resources/_gen/assets/scss/css/base.scss_3b33337114e481782feeb60752452e17.json:1]: Agreed - handled in 49dff0e87616ad887c1eb6c69785d1f57cc3a518

**Review by dreamPathsProjekt — APPROVED** [review]

**Pull request merged**